### PR TITLE
Fix-Improve Project UI 

### PR DESCRIPTION
**Pull request by archikirar30**
## Please, go through these steps before you submit a PR.

- [x] My Pod Leader knows I'm working on this Pull Request
- [x] I've explained what the Pull Request is adding.
- [x] I've explained why this is important.

**Fixed Issue #6** 
page which having bug - `http://127.0.0.1:4000/projects/project` and `http://127.0.0.1:4000/`

**Screenshot of UI after resolving  bugs-**
**Web View**
- Home page
`http://127.0.0.1:4000/`
image

- Project Page
`http://127.0.0.1:4000/projects/projec`
image

**Mobile view -** 
- Home page 
image

- Project Page
image

### CODE REVIEW

**Comment by archikirar30** [comment]
heyy @akshgpt7 , me and @jessielin34  have fixed issue #6  , Please review it. Thanks

File: Gemfile.lock [hunk 1]
```diff
@@ -1,18 +1,18 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    addressable (2.8.0)
-      public_suffix (>= 2.0.2, < 5.0)
+    addressable (2.8.3)
```
Comment (akshgpt7) [inline Gemfile.lock]: Please omit the Gemfile.lock from the PR, you don't need to commit it.

**Review by akshgpt7 — COMMENTED** [review]
Looks pretty good. Can you just:
1. Center the images inside the projects cards.
2. Write a name for the "Developer" field.

File: Gemfile.lock (hunk 1 above)
Comment (archikirar30) [reply Gemfile.lock]: @akshgpt7 can you let me know who are the developer of these specific project?

**Comment by jessielin34** [comment]
Hi Aksh,We actually were confused what to put for the developer part because we also don’t know who the developers were so we just put a generic “developer” there.Thank you,Jessie LinOn Apr 11, 2023, at 9:41 AM, Archi kirar ***@***.***> wrote:﻿
@archikirar30 commented on this pull request.



In Gemfile.lock:
> @@ -1,18 +1,18 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    addressable (2.8.0)
-      public_suffix (>= 2.0.2, < 5.0)
+    addressable (2.8.3)

@akshgpt7 can you let me know who are the developer of these specific project?

**Comment by archikirar30** [comment]
Hello @akshgpt7 I have done all the changes which you have mentioned except developer one, Please review once and let me know about the developer of these projects. Thank You

**Comment by archikirar30** [comment]
Hi, @akshgpt7 can you review it once, I have done all the necessary changes which is required and even there is no merge conflicts.

**Pull request closed without merging**